continuousintegration.py

```diff
@@ -24,7 +24,9 @@ def watch_ci():
     s.listen(10)
     print 'listening for CI changes'
 
-    while 1:
+    api_url = 'https://api.github.com/repos/Charcoal-SE/SmokeDetector/'
```

Comment:
Don't do this, it looks really confusing to me. I really have a hard time parsing the `{0}` stuff.